db/index.js

```diff
@@ -0,0 +1,20 @@
+// var AWS = require('aws-sdk');
+// var s3 = new AWS.S3();
+var faker = require('faker');
+var db = require('./index.js')
```

Comment:
the index is solid, but maybe the db = require line is unnecessary because it's self referential?  Otherwise it's nice and clear.